packages/beacon-node/src/api/impl/validator/index.ts

```diff
@@ -887,13 +888,17 @@ export function getValidatorApi(
         opts
       );
 
-      if (opts.blindedLocal === true && ForkSeq[meta.version] >= ForkSeq.bellatrix) {
+      if (
+        opts.blindedLocal === true &&
+        ForkSeq[meta.version] >= ForkSeq.bellatrix &&
+        ForkSeq[meta.version] < ForkSeq.gloas
+      ) {
```

Comment:
to make it a bit easier to read

```suggestion
      const fork = ForkSeq[meta.version];

      if (opts.blindedLocal === true && fork >= ForkSeq.bellatrix && fork < ForkSeq.gloas) {
```